scrapy/utils/gz.py

```diff
@@ -51,8 +51,12 @@ def gunzip(data):
     return output
 
 _is_gzipped_re = re.compile(br'^application/(x-)?gzip\b', re.I)
+_is_octetstream_re = re.compile(br'^(application|binary)/octet-stream\b', re.I)
```

Comment:
Ah right, _is_gzipped can also be written this way. I'd even drop 'is not None' checks. From https://docs.python.org/3.5/library/re.html#match-objects: 

> Match objects always have a boolean value of True

There is an example in stdlib docs:

```py
match = re.search(pattern, string)
if match:
    process(match)
```